backend/api/server/router/failure/failure.go

```diff
@@ -30,15 +30,24 @@ func (f *failureRouter) createFailure(ctx context.Context, w http.ResponseWriter
 		})
 	}
 
+	jiraBug, err := f.Storage.GetJiraBug(fr.JiraKey)
+	if err != nil {
+		f.Logger.Sugar().Warnf("Failed to get jira :", err)
+	}
+
 	jiraStatus, err := f.Storage.GetJiraStatus(fr.JiraKey)
 	if err != nil {
 		f.Logger.Sugar().Warnf("Failed to get jira status:", err)
 	}
 
 	err = f.Storage.CreateFailure(failureV1Alpha1.Failure{
-		JiraKey:      fr.JiraKey,
-		JiraStatus:   jiraStatus,
-		ErrorMessage: fr.ErrorMessage,
+		JiraKey:       fr.JiraKey,
+		TitleFromJira: jiraBug.Summary,
+		JiraStatus:    jiraStatus,
```

Comment:
Since now we are getting the jira bug, we can delete `GetJiraStatus` call

```suggestion
JiraStatus: jiraBug.Status,
```